src/Toml/Printer.hs

```diff
@@ -171,13 +173,17 @@ prettyTableArrays options i pref = mapOrdered arrText options
 
 -- Returns an indentation prefix
 tabWith :: PrintOptions -> Int -> Text
-tabWith options n = Text.replicate (n * indent options) " "
+tabWith options n = Text.replicate (n * printOptionsIndent options) " "
```

Comment:
We are using `RecordWildCards` so such code usually can be written a little bit nicer 🙂 I think it looks a bit cleaner.
```suggestion
tabWith PrintOptions{..} n = Text.replicate (n * printOptionsIndent) " "
```